site/docs/getting-started/design.mdx

```diff
@@ -1,15 +1,18 @@
 ---
 sidebar_label: Design
+hide_table_of_contents: true
 ---
 
+import Content from "../../src/components/content/Content.tsx";
+
 # Getting Started for designers
```

Comment:
```suggestion
# Designing with Salt
```

Reply:
the design page will be done separately :)